helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -352,6 +434,10 @@ public String getPath() {
   }
 
   public void enqueueTask(NotificationContext changeContext) throws Exception {
+    if (_periodicTriggerExecutor != null) {
```

Comment:
This check is no longer valid. We should use the interval or just use a boolean (_periodicTriggerEnabled) so it is clearer